INSTALL.md

```diff
@@ -55,10 +53,10 @@ To compile from source, open a **Terminal (Linux/MacOS)**, the **MSVC Tools Comm
    ```
    mkdir build
    cd build
-   cmake -DWITH_XC_ALL=ON ..
+   cmake ..
    make
```

Comment:
Or:

```suggestion
   cmake -B ./build
   cmake --build ./build
```

This way it also works with alternative "generators", such as Ninja instead of Make.